presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java

```diff
@@ -245,7 +245,7 @@ private static void compactTableAndWait(CompactionMode compactMode, String table
     private static void tryCompactingTable(CompactionMode compactMode, String tableName, String partitionString, Duration timeout)
             throws TimeoutException
     {
-        long beforeCompactionStart = Instant.now().getEpochSecond();
+        long beforeCompactionStart = Instant.now().getEpochSecond() * 1000;
```

Comment:
Can you pass Instant to `getTableCompactions` and do the conversion to low-level unit internally. It is implementation detail that in SQL query to get transactions we need milliseconds.

Reply:
Sure, I can